### ISSUE TITLE
TrailingSemicolon crashes when followed by a comment with a semicolon

### ISSUE THREAD

**Issue by mahtd** [brigade/scss-lint#574]
The following inputs crash scss-lint. There is no crash if you add the missing semicolon or remove the semicolon in the comment

```scss
@import 'foo'
//@import 'bar';
```

```scss
$foo: bar // comment;
```

Stack trace

```
SCSSLint::Linter::TrailingSemicolon raised unexpected error linting file testcase.scss: 'undefined method `[]' for nil:NilClass'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:71:in `character_at'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter/trailing_semicolon.rb:60:in `ends_with_multiple_semicolons?'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter/trailing_semicolon.rb:47:in `check_semicolon'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter/trailing_semicolon.rb:33:in `visit_import'
/Library/Ruby/Gems/2.0.0/gems/sass-3.4.16/lib/sass/tree/visitors/base.rb:36:in `visit'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:131:in `visit'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:142:in `block in visit_children'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:140:in `each'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:140:in `visit_children'
/Library/Ruby/Gems/2.0.0/gems/sass-3.4.16/lib/sass/tree/visitors/base.rb:38:in `visit'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:131:in `visit'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/linter.rb:25:in `run'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:49:in `run_linter'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:30:in `block in find_lints'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:28:in `each'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:28:in `find_lints'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:18:in `block in run'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:17:in `each'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/runner.rb:17:in `run'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/cli.rb:58:in `scan_for_lints'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/cli.rb:51:in `act_on_options'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/lib/scss_lint/cli.rb:28:in `run'
/Library/Ruby/Gems/2.0.0/gems/scss_lint-0.41.0/bin/scss-lint:6:in `<top (required)>'
/usr/bin/scss-lint:23:in `load'
/usr/bin/scss-lint:23:in `<main>'
```

Versions

```
- SCSS-Lint version: 0.41.0
- Sass version: 3.4.16
- Ruby version: 2.0.0
```

**Comment by sds** [comment]
Fixed in #583. Thanks for the report!
